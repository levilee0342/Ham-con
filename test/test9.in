-4823
-25928
-15294
-8397
