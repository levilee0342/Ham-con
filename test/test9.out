-4823
-25928